### PR TITLE
fix(storybook): preview config

### PR DESCRIPTION
**Pull request by gepifanio**
- Added lang to HTML
- Removed modal code that adds it to every page
- Added empty story for testing purposes.

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 342f4fe35051e8c1d7c1a521feadeacab3144bdb

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/609c747d30506200087dd0b9

File: components/test/stories/index.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { storiesOf } from '@storybook/vue';
+import { createStory } from 'storybook/utils';
+
+import Story1 from './Story1.vue';
+
+
+/* ##Import story component here */
+
+storiesOf('Empty Story', module)
```
Comment (jaredhenderson) [inline components/test/stories/index.js:9]: - [ ] Can you give these more meaningful names - 'Accessibility' category, and name the story 'Hidden page resources'

**Review by jaredhenderson — CHANGES_REQUESTED** [review]
Changes look good, just need to update the story name/category so it's clearer what the purpose is

**Review by jaredhenderson — APPROVED** [review]
👍 looks good. We should flag in the release notes that the Modal has changed and should be monitored

**Pull request merged**